.golangci.yaml

```diff
@@ -0,0 +1,31 @@
+run:
+  timeout: 1m
+  tests: true
+
+linters:
+  enable:
+    - asciicheck
+    - forcetypeassert
+    - gocritic
+    - godot
+    - gofmt
+    - goimports
+    - misspell
+    - stylecheck
+
+issues:
+  exclude-use-default: false
+  max-issues-per-linter: 0
+  max-same-issues: 10
+
+linters-settings: # please keep this alphabetized
+  gocritic:
+    enabled-checks:
+      - equalFold
+      - boolExprSimplify
+  staticcheck:
+    checks:
+      - "all"
+  stylecheck:
+    checks:
+      - "ST1019"   # Importing the same package multiple times - enabled in golangci-strict.yaml.
```

Comment:
I don't think we need any of these special `linters-settings`.

Reply:
I suppose we should? Kubernetes had to choose linter rules to match the code. In this much smaller project it is feasible to choose linter rules, then make the code comply.

Reply:
After poking around - I disabled one check for gocritic - singleCaseSwitch, it is intended to be a single case switch where it complains due to potentially more resource types implementation.

Reply:
Rest of the config was wiped, good to go IMHO